integration/proxy_test.go

```diff
@@ -241,6 +247,12 @@ func runTestGetManifestAndConfig(p *proxy, img string) error {
 	}
 	imgid := uint32(imgidv)
 
+  info, err := p.callNoFd("LayerInfoJSON", []interface{}{imgid})
+  if err == nil {
+    return err
+  }
+  fmt.Printf("Field: %+v\n", info)
```

Comment:
`fmt.Printf` should not be used in tests. And this is not testing much.

(Ultimately, I guess the test would be to verify that the returned info is actually usable for pulling, OTOH that might be too costly.)